Makefile

```diff
@@ -60,7 +60,7 @@ static: build
 
 .PHONY: unit
 unit:
-	$(GO) test -coverprofile=coverage.out $(SPECIFIC_UNIT_TEST) $(TAGS) $(TEST_RACE) -count=1 ./pkg/... ./alpha/...
+	$(GO) test -coverprofile=coverage.out -coverpkg=./... $(SPECIFIC_UNIT_TEST) $(TAGS) $(TEST_RACE) -count=1 ./pkg/... ./alpha/...
```

Comment:
This change means we now consider cross-package coverage. (i.e. the tests in `pkg/server/server_test.go` can contribute to coverage in `pkg/registry/query.go`)

This very likely means there's an artificial increase in the repo-wide coverage, but I did my best to add specific unit tests related to this PR in `pkg/registry/query_test.go` to make sure the new code paths are relatively well-covered.